src/lightstep-opentelemetry-launcher-node.ts

```diff
@@ -87,7 +88,14 @@ function setupLogger(
         process.env.OTEL_LOG_LEVEL.toUpperCase() as keyof typeof LogLevel
       ] ?? logLevel;
   }
-  return new ConsoleLogger(logLevel);
+
+  const logger = new ConsoleLogger(logLevel);
+
+  if (logLevel == LogLevel.DEBUG && !config.logger) {
```

Comment:
consider strict comparison `===` 
Why only for DEBUG you are attaching logger, I think it might be a bit misleading for a user why adding a logger to config will have no effect

Reply:
I didn't want to force a ConsoleLogger at info level on all users by default. For "debug mode" it made sense to attach the ConsoleLogger if a user did not provide one (otherwise the debug logs go to the NoopLogger). @codeboten, thoughts?

Reply:
The default value for Python and Go is to log errors. Really the only difference as of now is that debug will print out the config information.

Reply:
Framed differently. Should the default logger be a ConsoleLogger at the default value of OTEL_LOG_LEVEL? Right now the default logger is a NoopLogger and OTEL_LOG_LEVEL defaults to info.

Reply:
Tentatively, I think I'll go with this plan. I'll set the default logger to a ConsoleLogger at info level. For the purposes of getting information into Lightstep, this seems reasonable. Users can provide their own Logger instance or change the log level if this doesn't suite their needs.

Reply:
I think I missed one thing here thinking that the logger is coming from user and will only be used if you setup DEBUG log level which is not the case. I would leave it as it is as in production you don't want to use debug and console logger for sure.